src/databricks/labs/ucx/hive_metastore/view_migrate.py

```diff
@@ -41,7 +41,7 @@ def _view_dependencies(self):
             yield TableView("hive_metastore", src_db, old_table.name)
 
     def sql_migrate_view(self, index: MigrationIndex) -> str:
-        from_table = FromTable(index, use_schema=self.src.database)
+        from_table = FromTable(index)
```

Comment:
this breaks views migration. i wonder why `make test` didn't fail. 
```suggestion
        from_table = FromTable(index, CurrentSessionState(self.src.database))
```

Reply:
Good spot

Reply:
I will check the tests and see if I can add one that would point this out

Reply:
There are no tests that would have caught that. Given the session state is now mandatory, perhaps there is no need for one?